components/licence-title.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function LicenceTitle ({
+	displayName = '',
+	isTrial = false,
+	isB2cPartnershipLicence = false,
+}){
+	if(isB2cPartnershipLicence || isTrial) {
+		return (
+			<h1 className="ncf__header ncf__center">
+				{
+					(displayName !== '') && (`${displayName} | `)
+				}
+				{
+					isB2cPartnershipLicence
+						? 'Welcome to the Financial Times'
+						: 'Start your free trial'
+				}
+			</h1>
+		);
+	}
+	{
+		return renderB2BTitle(displayName);
+	}
+
+}
+
+function renderB2BTitle (displayName) {
+	const companyName = (displayName !== '') ? displayName : 'Your company';
```

Comment:
this is fine, I would personally go with the following.
```suggestion
	const companyName = displayName ? displayName : 'Your company';
```
I know we expect a string but this won't change anything for empty strings and if we somehow get a 0 or null or false we're  covered.
It's also simpler to write :)